tests/dummy/config/environment.js

```diff
@@ -24,7 +24,8 @@ module.exports = function(environment) {
     cognito: {
       poolId: 'us-east-1_TEST',
       clientId: 'TEST',
-      autoRefreshSession: true
+      autoRefreshSession: true,
+      authenticationFlowType: 'USER_SRP_AUTH'
```

Comment:
Good point! 
Yes, it will fail the config tests in authenticator and service. 
How about we test them against ENV config values instead of existing hardcoded value? 
Happy to send in a PR if that sounds okay.

Reply:
I'm not sure I understand you. The tests all currently pass, but what I'm asking is when you run the dummy app itself (`ember serve` in the addon). This changes the dummy app to use USER_SRP_AUTH authentication flow type, but not everyone would use that. Ideally we'd want to have the dummy app be able to work with either authentication flow types.

Reply:
Oops, sorry I mistook with tests in dummy app!

Nope, the dummy app won't break. If `authenticationFlowType` is set in config (`USER_SRP_AUTH` or `USER_PASSWORD_AUTH`), it uses that. Otherwise, it uses the default auth flow type (`USER_SRP_AUTH`). So it will work with/without `authenticationFlowType`. 

But tests need a bit of change like mentioned above. Since it is testing against hardcoded value. 
https://github.com/paulcwatts/ember-cognito/pull/92/files#diff-b2f1dd3e76b43a24a389f9fffad10f8aR32
I can update my PR to test against config values.

Reply:
Ah, I understand now. Thanks! Feel free to change the tests to use ENV values or not. I'll do a bit of testing with the branch, but otherwise it looks good to go.